src/questions/funcTypeQuestions.tsx

```diff
@@ -0,0 +1,196 @@
+    type: QuestionType.Minifigures,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: (
+      <span>
+        Must {segmentQuestion("BA")} {strs.congruent} {segmentQuestion("CB")}?
+      </span>
+    ),
+    answers: ["Yes", "No"],
+    type: QuestionType.DiagramState,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: (
+      <span>
+        Must {segmentQuestion("CB")} {strs.congruent} {segmentQuestion("AD")}?
+      </span>
+    ),
+    answers: ["Yes", "No"],
+    type: QuestionType.DiagramState,
+  },
+];
+
+export const checkingProof2: Question[] = [
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: "Is Congruent Adjacent Angles correctly applied?",
+    answers: ["Yes", "No"],
+    type: QuestionType.Minifigures,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: `Must ${strs.angle}DAB ${strs.congruent} ${strs.angle}BDC?`,
+    answers: ["Yes", "No"],
+    type: QuestionType.DiagramState,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: (
+      <span>
+        Must {segmentQuestion("KL")} {strs.congruent} {segmentQuestion("MK")}?
+      </span>
+    ),
+    answers: ["Yes", "No"],
+    type: QuestionType.DiagramState,
+  },
+];
+
+export const checkingProof3: Question[] = [
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: "Is HL congruence correctly applied?",
+    answers: ["Yes", "No"],
+    type: QuestionType.Minifigures,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: "Is Def. Rectangle correctly applied?",
+    answers: ["Yes", "No"],
+    type: QuestionType.Minifigures,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: (
+      <span>
+        Must {segmentQuestion("KN")} {strs.congruent} {segmentQuestion("ML")}?
+      </span>
+    ),
+    answers: ["Yes", "No"],
+    type: QuestionType.DiagramState,
+  },
+];
+
+export const completeProof1: Question[] = [
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: "Is Alternative Interior Angles correctly applied?",
+    answers: ["Yes", "No"],
+    type: QuestionType.Minifigures,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt:
+      "Is there enough information to apply Vertical Angles between step 1 and 2?",
+    answers: ["Yes", "No"],
+    type: QuestionType.ReliesOn,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt:
+      "Is there enough information to apply SAS triangle congruence between step 2 and 3",
+    answers: ["Yes", "No"],
+    type: QuestionType.ReliesOn,
+  },
+];
+
+export const completeProof2: Question[] = [
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: "Is Def. Perpendicular correctly applied?",
+    answers: ["Yes", "No"],
+    type: QuestionType.Minifigures,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt: "Is midpoint correctly applied?",
+    answers: ["Yes", "No"],
+    type: QuestionType.Minifigures,
+  },
+  {
+    fullScaffold: "Placeholder: ",
+    prompt:
+      "Is there enough information to apply Congruent Adjacent Angles at this point?",
```

Comment:
hmm, some of these questions are written assuming that the student is presented a specific state... these questions should probably be reworded to say "is there enough information to apply _____ between step A and step B?"

Reply:
This one should become "Is there enough information to apply Congruent Adjacent Angles at step 3?"